### ISSUE TITLE
Ability to have mulitple planning applications within an issue

### ISSUE THREAD

**Issue by roxannedebeaux** [cyclestreets/cyclescape#512]
Planning applications create new issues, however many of them are related to the same issue. It makes it easier to keep on top of the planning applications if they can be kept under one issue. Also helps to see the historical information about previous decisions.

**Comment by mvl22** [comment]
Yes, it's become clear that a planning application sometimes needs to be clonable to a new thread rather than a new issue. We'll need to think how to handle this.

PS The user interface for the relationship between issues/threads is due to change, following discussions on Thursday.

**Comment by roxannedebeaux** [comment]
That's great.

If you would like to work through it with a user then I would be happy to
share my views.

I've been thinking a lot about the structure/hierarchy of this and how it
could work more efficiently.



Kind Regards,

Roxanne De Beaux
Cycling Campaign Officer

*Cambridge Cycling Campaign*
Charity, no. 1138090

t: @camcycle     w: www.camcycle.org.uk/     e: contact@camcycle.org.uk
Join the Campaign at: www.camcycle.org.uk/membership/join/
*Donate to support the work we do. Text "CYCC11 £5" to 70070*
